services/IoT/applications/libraries/java.md

```diff
@@ -136,7 +136,7 @@ By default, applications subscribe to all events from all connected devices. Use
     myClient.subscribeToDeviceEvents("iotsample-ardunio", "00aabbccddee", "myEvent");
     myClient.subscribeToDeviceEvents("iotsample-ardunio", "10aabbccddee", "myEvent");
 ```
-### Subscribing to events published by a device in JSON format
+### Subscribing to events that are published in JSON format
```

Comment:
the phrase "by a device" is implied by "published" - the conceptual info has already made clear that devices do the publishing. The phrase was making "in JSON format" dangle at the end of the sentence, which made it seem like the device is in JSON format.